Sources/SentryCrash/Recording/Monitors/SentryCrashMonitorContext.h

```diff
@@ -220,9 +220,6 @@ typedef struct SentryCrash_MonitorContext {
         const char *reason;
     } ZombieException;
 
-    /** Full path to the console log, if any. */
-    const char *consoleLogPath;
```

Comment:
This is only ever used in one place, and at one time in a process run. I just moved the access to be done directly if that spot.